tests/test_sample_raycluster_yamls.py

```diff
@@ -78,7 +78,7 @@ def parse_args():
             logger.info('[SKIP TEST %d] %s: %s', index, new_cr['name'], skip_tests[new_cr['name']])
             continue
         logger.info('[TEST %d]: %s', index, new_cr['name'])
-        addEvent = RayClusterAddCREvent(new_cr['cr'], [rs], 90, NAMESPACE, new_cr['path'])
+        addEvent = RayClusterAddCREvent(new_cr['cr'], [rs], 180, NAMESPACE, new_cr['path'])
```

Comment:
I found the RayClusterAddCREvent flaky due to its short timeout. Double it to 180 seconds to reduce flakiness.

Reply:
We should not increase the timeout here if we don't know why the test can't finish in 90 seconds. This may obscure the actual root cause. There might be something happening under the hood, because in most cases, the failure is related to `ray-cluster.external-redis-uri.yaml` rather than occurring randomly across multiple tests. We don’t need to investigate the actual root cause in this PR, but we should still keep the timeout at 90 seconds until we understand what happened.